ui/component/core/src/index.ts

```diff
@@ -891,13 +893,18 @@ export class Manager implements EventProviderFactory {
                             this._keycloak!.clearToken();
                             this._keycloak!.login();
                         } else {
-                            console.error("Something went wrong reaching the keycloak server. Not redirecting.")
+                            console.error("Something went wrong reaching the keycloak server. Not redirecting.");
+                            this._emitEvent(OREvent.AUTH_REFRESH_FAILED);
```

Comment:
Event should be emitted as soon as `onAuthRefreshError` is called not wait for fetch to timeout or fail